src/lib.rs

```diff
@@ -437,24 +437,24 @@ pub mod lib {
 }
 
 pub use self::bits::*;
-pub use self::internal::*;
+pub use self::parser::*;
 pub use self::traits::*;
 
 pub use self::str::*;
 
 #[macro_use]
 pub mod error;
 
-mod internal;
-mod traits;
 pub mod combinator;
+mod parser;
+mod traits;
```

Comment:
Can you split these rustfmt changes (reordering of `mod {traits,sequence,bytes}`) to a separate commit? Otherwise git blame becomes unnecessarily noisy.

Reply:
Done.